examples/ghost/ghost-aws.yaml

```diff
@@ -0,0 +1,43 @@
+name: ghost
+org_name: <<org_name::org_name::Your Organization Name>>
+providers:
+  aws:
+    region: <<region::aws_region::Your AWS Region>>
+    account_id: <<account_id::aws_account_id::Your AWS Account ID>>
+modules:
+  - type: base
+  - type: dns
+    domain: <<domain::string::The dns domain for your Ghost deployment>>
+    delegated: false
+  - type: k8s-cluster
+  - type: k8s-base
+  - type: mysql
+    name: mysql
+    safety: false
+  - type: k8s-service
+    env_vars:
+      - name: database__client
+        value: "mysql"
+      - name: url
+        value: "https://${{module.dns.domain}}/dashboard"
```

Comment:
would that work even if the user only does the section `# Steps to deploy`?

Reply:
Yup this works. they can still write any random domain in the domain section and not delegate. and then just use the load balancer url